bundles/com.espressif.idf.serial.monitor/build.properties

```diff
@@ -18,4 +18,6 @@ bin.includes = META-INF/,\
                lib/log4j-1.2.14.jar,\
                lib/pty4j-0.8.6.jar,\
                lib/purejavacomm-0.0.11.1.jar,\
-               OSGI-INF/l10n/bundle.properties
+               OSGI-INF/l10n/bundle.properties,\
+               lib/Java-WebSocket-1.5.2.jar,\
+               lib/slf4j.api-1.6.1.jar
```

Comment:
We could remove slf4j if not used

Reply:
slf4j is required for the java-websocket jar we are using without it the lib doesn't work